Doc/whatsnew/3.10.rst

```diff
@@ -109,6 +109,12 @@ base64
 Add :func:`base64.b32hexencode` and :func:`base64.b32hexdecode` to support the
 Base32 Encoding with Extended Hex Alphabet.
 
+codecs
+------
+
+Add :func:`codecs.unregister` and :c:func:`PyCodec_Unregister` to unregister
```

Comment:
New C API functions should be documented in the C API section:
https://docs.python.org/dev/whatsnew/3.10.html#id1